templates/vue/src/components/NodeModal.vue

```diff
@@ -270,22 +276,31 @@
 
 <script>
 import Helpers from "../utils/Helpers"
-import Combobox from "./Combobox"
+import { tydeTypes } from "../utils/constants"
 import QuizModal from "./node-modal/QuizModal"
+import TydeTypeInput from "./node-modal/TydeTypeInput"
+import { mapGetters } from "vuex"
+import Combobox from "./Combobox"
```

Comment:
To minimize merge conflicts with master, please don't move lines if not necessary (unless you also move it on master)